tap-snapshots/test/v8-to-istanbul.js.test.cjs

```diff
@@ -279,8 +279,8 @@ Object {
       "line": 35,
       "loc": Object {
         "end": Object {
-          "column": 0,
-          "line": 37,
+          "column": 29,
```

Comment:
The column changing significantly concerns me a little bit, because I think it suggests we identified a different statement, this is where source maps can really start to behave drastically different.

A good TypeScript codebase to test on is this one:

https://github.com/googleapis/release-please

and this one:

https://github.com/yargs/yargs